llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -797,7 +797,7 @@ def HasV8_9aOps : Architecture64<8, 9, "a", "v8.9a",
   !listconcat(HasV8_8aOps.DefaultExts, [FeatureSPECRES2, FeatureCSSC,
     FeatureRASv2])>;
 def HasV9_0aOps : Architecture64<9, 0, "a", "v9a",
-  [HasV8_5aOps, FeatureMEC],
```

Comment:
Can you move this into the additional defaultexts, instead of removing it?

Reply:
Sure, but it belongs on v9.2, not here.